applications/nrf_desktop/description.rst

```diff
@@ -756,6 +816,14 @@ This system state LED is kept lit when the device is active.
       **LED1** is used for the system state indication.
       It is located in the bottom right corner of the dongle, next to the USB connector.
 
+   .. tab:: nRF54H DK
```

Comment:
The LED indiication tab uses the "nRF54H DK" category while the peer control button uses "nRF54 DK"

Reply:
Thanks. Updated on whole page